tests/unit/channel_tests.py

```diff
@@ -590,6 +601,7 @@ def test_confirm_delivery_with_bad_callback_raises_value_error(self):
                           'bad-callback')
 
     def test_confirm_delivery_raises_channel_closed(self):
+        self.obj._closing_code_and_text = (123, "here is why it was closed")
```

Comment:
Need to set these up since `Channel. _closing_code_and_text` now defaults to `None`.